packages/byods/jest.config.js

```diff
@@ -0,0 +1,43 @@
+export default {
+  preset: 'ts-jest',
+  rootDir: './',
+  setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
+  testEnvironment: 'jsdom',
```

Comment:
This should be 'node'
See: https://jestjs.io/docs/configuration#testenvironment-string

Reply:
I did use 'node' but while running the test I was receiving below error:

Reply:
Please go ahead and remove the import. We don't need it.